recycler/src/main/java/com/elpassion/android/commons/recycler/RecyclerViewAdapterCompositor.kt

```diff
@@ -0,0 +1,33 @@
+package com.elpassion.android.commons.recycler
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.elpassion.android.commons.recycler.components.ItemsStrategy
+import com.elpassion.android.commons.recycler.components.base.ItemAdapter
+
+class RecyclerViewAdapterCompositor<T : ItemAdapter<out RecyclerView.ViewHolder>>(private val itemsStrategy: ItemsStrategy<T>,
```

Comment:
Constructor is a little hard to read. Split this line after `(` for readability and then apply _auto format_.